core/arch/arm32/plat-stm/system_config.in

```diff
@@ -40,6 +40,7 @@ endif
 # Common values
 
 CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= 1
+CFG_TEE_CORE_NO_TA_TRACE ?= n
```

Comment:
Just define it in mk/config.mk (this is the global, default config for all platforms).